terraso_backend/apps/story_map/views.py

```diff
@@ -102,6 +110,15 @@ def post(self, request, **kwargs):
 
         new_config = json.loads(form_data["configuration"])
 
+        if invalid_media_type(new_config):
+            logger.info("Warning: invalid media type")
+            error_message = ErrorMessage(
+                code="Invalid Media Type",
+                context=ErrorContext(model="StoryMap", field=NON_FIELD_ERRORS)
+            )
+            return JsonResponse({"errors": [{
+                "message": [asdict(error_message)]}]}, status=400)
```

Comment:
question: why is this called twice?

Reply:
The first time this is called is validating the media type added is correct, and the second time this is called is for validating if the new media being uploaded in replace of the old one is correct